htdocs/inc/xajax/techtree.xajax.php

```diff
@@ -11,33 +11,33 @@ function reqInfo($id,$cat='b')
 
 
 	// Load items
-	$bures = dbquery("SELECT building_id,building_name FROM buildings WHERE building_show=1;");
-	while ($buarr = mysql_fetch_array($bures))
+	$res = dbquery("SELECT building_id,building_name FROM buildings WHERE building_show=1;");
+	while ($arr = mysql_fetch_array($res))
 	{
-		$bu_name[$buarr['building_id']]=$buarr['building_name'];
+		$bu_name[$arr['building_id']]=$arr['building_name'];
 	}		
-	$teres = dbquery("SELECT tech_id,tech_name FROM technologies WHERE tech_show=1;");
-	while ($tearr = mysql_fetch_array($teres))
+	$res = dbquery("SELECT tech_id,tech_name FROM technologies WHERE tech_show=1;");
+	while ($arr = mysql_fetch_array($res))
 	{
-		$te_name[$tearr['tech_id']]=$tearr['tech_name'];
+		$te_name[$arr['tech_id']]=$arr['tech_name'];
 	}	
 
-	$teres = dbquery("SELECT ship_id,ship_name FROM ships WHERE ship_show=1 AND special_ship=0;");
-	while ($tearr = mysql_fetch_array($teres))
+	$res = dbquery("SELECT ship_id,ship_name FROM ships WHERE ship_show=1 AND special_ship=0;");
+	while ($arr = mysql_fetch_array($res))
 	{
-		$sh_name[$tearr['ship_id']]=$tearr['ship_name'];
+		$sh_name[$arr['ship_id']]=$arr['ship_name'];
 	}	
 	
-	$teres = dbquery("SELECT def_id,def_name FROM defense WHERE def_show=1;");
-	while ($tearr = mysql_fetch_array($teres))
+	$res = dbquery("SELECT def_id,def_name FROM defense WHERE def_show=1;");
+	while ($arr = mysql_fetch_array($res))
 	{
-		$de_name[$tearr['def_id']]=$tearr['def_name'];
+		$de_name[$arr['def_id']]=$arr['def_name'];
 	}		
 	
-	$teres = dbquery("SELECT missile_id,missile_name FROM missiles WHERE missile_show=1;");
-	while ($tearr = mysql_fetch_array($teres))
+	$res = dbquery("SELECT missile_id,missile_name FROM missiles WHERE missile_show=1;");
+	while ($arr = mysql_fetch_array($res))
 	{
-		$m_name[$tearr['missile_id']]=$tearr['missile_name'];
+		$m_name[$arr['missile_id']]=$arr['missile_name'];
```

Comment:
Can you please revert the renaming changes? Having different variables can help automated tools find bugs. If you have naming or formatting changes you would like to apply if you could create separate pull requests for them. Makes it easier to review.